src/wkt/src/wrappers.rs

```diff
@@ -113,17 +113,43 @@ pub type StringValue = String;
 /// The JSON representation for `BytesValue` is JSON string.
 pub type BytesValue = bytes::Bytes;
 
+pub(crate) struct ProtobufTypeSerializer<T> {
+    _phantom: std::marker::PhantomData<T>,
+}
+
+impl<T> ProtobufTypeSerializer<T> {
+    pub fn new() -> Self {
+        Self {
+            _phantom: std::marker::PhantomData,
+        }
+    }
+}
+
+impl<T> crate::message::MessageSerializer<T> for ProtobufTypeSerializer<T>
```

Comment:
This one can also be a `crate::message::ValueSerializer<T>`

Reply:
Indeed. Done.